src/wp-includes/cache.php

```diff
@@ -132,6 +132,36 @@ function wp_cache_set_multiple( array $data, $group = '', $expire = 0 ) {
 	return $wp_object_cache->set_multiple( $data, $group, $expire );
 }
 
+/**
+ * Removes all cache items in a group.
+ *
+ * @param string|array $group name(s) of group to remove from cache.
+ *
+ * @return bool|Array True or array of bool in array passed on success, false on failure group not found.
+ * @global WP_Object_Cache $wp_object_cache Object cache global instance.
+ *
+ * @since 6.0.0
+ *
+ * @see WP_Object_Cache::flush_group()
+ */
+function wp_cache_flush_group( $group ) {
+	global $wp_object_cache;
+
+	// if group is an array loop and call each key in the array
+	if ( is_array( $group ) ) {
+		$result   = false;
+		$result[] = array_map( 'wp_cache_flush_group', array_values( $group ) );
+
+		return $result;
+	}
+
+	if ( method_exists( $wp_object_cache, 'flush_group' ) ) {
+		return $wp_object_cache->flush_group( $group );
+	}
+
+	return false;
```

Comment:
The `flush_group()` method will always exist, since plugin supply their own `wp_cache_flush_group()` function, or fallback to the compat method.

```suggestion
	return $wp_object_cache->flush_group( $group );
```